javascript/code-challenges/array-insert-shift/README.md

```diff
@@ -0,0 +1,2 @@
+### Code Challenge White Board
+![WhiteBoard Solution](array-insert-shift.jpg)
```

Comment:
Just change the title to the name of the challenge. Along with adding a description, and the approach and efficiency(What approach did you take?)